.circleci/config.yml

```diff
@@ -283,6 +283,10 @@ commands:
     steps:
       - restore_cache:
           key: v5-ui-cache-{{ .Branch }}-{{ checksum "~/workbench/ui/yarn.lock" }}
+      - gcloud-auth-login
+      - run:
```

Comment:
I understand why this step is needed, but please add a comment or step description for future reference